services/ProductRegistrationService.php

```diff
@@ -88,6 +88,8 @@ private function optInStrategy($email) {
         curl_setopt($curl, CURLOPT_POST, true);
         curl_setopt($curl, CURLOPT_POSTFIELDS, http_build_query(array('email' => $email)));
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
Actually, may of found answer to my question here:
http://stackoverflow.com/questions/33795717/why-we-need-curlopt-ssl-verifypeer-in-windows